powerbi-docs/developer/visuals/capabilities.md

```diff
@@ -28,6 +29,82 @@ The root objects of a visual's capabilities are `dataRoles`, `dataViewMappings`,
 
 ```
 
+## Define the special permissions that your visual requires: privileges
+
+Privileges are the special operations required by your visual. It takes an array of `Privilege` objects, which defines all privilege properties. Following is the list of possible privileges:
+
+### General Privilege definition
+
+```json
+{
+    // string - The name of the privilege
+    "name"
+
+    // boolean - essentiality of the privilege. true - means, that the normal 
+    // visual functionality requires this privilege. false - it is not mandatory
+    // The property is optional and if missing - it considered to be false
+    "essential"
+    
+    // array of strings - privilege arguments as a list of strings
+    // The property is optional and if missing - it considered to be an empty array
+    "parameters"
+}
+```
+
+### Access external resources
+
+Visual going to access any external resource, must add a `WebAccess` privilege into capabilities section. The privilege can definition can contain an optional list of URLs the visual need to access in form of http://xyz.com or https://xyz.com. Every URL can include a wildcard to specify subdomains as well
+
+#### Example
+
+```json
+{
+    "name": "WebAccess",
+    "essential": true,
+    "parameters": [ "https://*.virtualearth.net" ]
+}
+```
+
+The above definition means, that the visual need to access any subdomain of the virtualearth.net via HTTPS protocol only and that this is essential for visual's normal work.
+
+### Access browser local storage
+
+Visual going to access browser local storage via [Local Storage API](./local-storage.md), must add a `LocalStorage` privilege into capabilities section.
```

Comment:
The visual will access the browser local storage via the [...

Reply:
Fixed.